$OpenBSD: patch-Makefile_am,v 1.3 2016/01/05 23:50:56 sthen Exp $

XXX Attempt to avoid an intermittent crash in ld.so when running vlc-cache-gen.
See https://marc.info/?t=145123666100001&r=1&w=2

The LD_PRELOAD alone seems to help i386 quite a lot, but doesn't do much
on amd64. But adding the LD_DEBUG=1 seems to improve things a lot there....

--- Makefile.am.orig	Thu Jul 16 03:14:03 2015
+++ Makefile.am	Thu Jul 16 03:13:29 2015
@@ -984,6 +984,7 @@ install-exec-hook:
 	if test "$(build)" = "$(host)"; then \
 		PATH="$(DESTDIR)$(bindir):$$PATH" \
 		LD_LIBRARY_PATH="$(DESTDIR)$(libdir):$$LD_LIBRARY_PATH" \
+		LD_DEBUG=1 LD_PRELOAD=${LOCALBASE}/lib/libgobject-2.0.so:${LOCALBASE}/lib/libnotify.so:${LOCALBASE}/lib/libgmodule-2.0.so \
 		"$(DESTDIR)$(vlclibdir)/vlc-cache-gen$(EXEEXT)" \
 			 "$(DESTDIR)$(vlclibdir)/plugins" ; \
 	else \
